index.js

```diff
@@ -22,6 +22,36 @@ const TREZOR_CONNECT_MANIFEST = {
   appUrl: 'https://metamask.io',
 };
 
+const oneKeySpecialVersion = 99;
+const oneKeyVendor = 'onekey.so';
+
+/**
+ * get the vendor name of the hardware wallet
+ * @param {object} features
+ * @returns {'onekey' | 'trezor'}
```

Comment:
Looks like based on the code that this should be:

``` suggestion
 * returns {'onekey' | 'trezor' | undefined}
```

Reply:
Done, I've updated the comments for this function.